### DISCUSSION TITLE
Change Font Size in a Label

### DISCUSSION THREAD

**Discussion by bensuffolk** [HASwitchPlate/openHASP#677] [Q&A]
Is it possible to change a Font Size in a label, like you can change the the colour with embedding a #.

I'd like to be able to use a button with a large Icon and then a smaller text description.

**Comment by fvanroie** [comment]
There is only one `text_font` for the label text. You can use a different `value_font` for an additional *floating text* or a different label for another font. However you can't mix fonts within the same text.

You can try to add a label to the button by setting its `parentid`, but that's going to be harder to position, I guess...

**Reply by bensuffolk** [reply]
A child label is a good idea. I’ll try that.
